### PR TITLE
recursive Methods: add new video that explanation recursion 

### PR DESCRIPTION
**Pull request by nmcev**
## Because

Added [this video](https://youtu.be/LteNqj4DFD8) making the recursion more easy to understand 

## This PR

- Add relative recursion video link to task two in assignment from [DevSage](https://youtu.be/LteNqj4DFD8)

## Issue

Closes #XXXXX

## Additional Information
the reason of this commit ["remove unwanted space this"](https://github.com/TheOdinProject/curriculum/commit/8f4c75a986d3cc89afd7720ff90d1798727a2a84)  is a space added in wrong so i fixed to following the style guide ,
this PR is relative to last PR as there are changes requested by @LeoKirasic,
@LeoKirasic  this is the PR that you request .

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: javascript/computer_science/recursive_methods.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ This section contains a general overview of topics that you will learn in this l
 
 <div class="lesson-content__panel" markdown="1">
   1. Read [this recursion resource](https://javascript.info/recursion) for a good intro to recursion.
-  2. Watch [this explanation of Recursion](https://www.youtube.com/watch?v=6oDQaB2one8) by Web Dev Simplified.
+  2. Watch [this explanation of Recursion](https://www.youtube.com/watch?v=6oDQaB2one8) by Web Dev Simplified and [this explanation of Recursion](https://youtu.be/LteNqj4DFD8) by DevSage.
```
Comment (rlmoser99) [inline javascript/computer_science/recursive_methods.md:28]: In watching both of these videos, I think the first example of the DevSage one is overkill. However, the second one has a great explanation. Therefore I would like to add a timestamp to this video (and remove the capitalization of Recursion). I was thinking something like this:

```suggestion
  2. Watch [this explanation of recursion](https://www.youtube.com/watch?v=6oDQaB2one8) by Web Dev Simplified and [this additional example of recursion](https://youtu.be/LteNqj4DFD8?t=340) by DevSage.
```

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
I've requested a small change. Feel free to take this suggestion and make additional changes. Or if you are happy with this suggestion, you can choose "commit suggestion".

**Comment by nmcev** [comment]
@rlmoser99  thank you for your time to review my PR 
and i see the suggestion that you offer was too good and i commit the changes

**Review by rlmoser99 — APPROVED** [review]

**Comment by rlmoser99** [comment]
Awesome! Thanks so much for taking the time to submit this PR!

**Comment by nmcev** [comment]
> Awesome! Thanks so much for taking the time to submit this PR!

thanks, my honour

**Pull request merged**